src/pipeline.py

```diff
@@ -129,12 +153,24 @@ def process_folder(self, in_folder, out_folder=None, suffix='',
         if suffix is None:
             suffix = ''
 
-        file_types = ('.LAS', '.las', '.LAZ', '.laz')
-
-        files = [f for f in in_folder.glob('*') if f.name.endswith(file_types)]
-
-        for file in tqdm(files, unit="file", disable=hide_progress):
-            # Load LAS file.
-            filename, extension = os.path.splitext(file)
-            outfile = filename + suffix + extension
+        logger.info('===== PIPELINE =====' +
+                    f'Processing folder {in_folder}, ' +
+                    f'writing results in {out_folder}.')
+
+        files = [f for f in in_folder.glob('*')
+                 if f.name.endswith(self.FILE_TYPES)
```

Comment:
Is het self.FILE_TYPES of FILE_TYPES?

Reply:
Het is `self.FILE_TYPES` of `Pipeline.FILE_TYPES` (class variable).

Reply:
Anders kan hij hem niet vinden :)